<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Naveen-Mac.local" ignored="0" name="Go Rest regression Test" tests="10" failures="1" timestamp="30 Aug 2019 03:12:51 GMT" time="91.272" errors="0">
  <testcase name="createUserAPIPOSTTest" time="47.133" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="49.344" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="54.341" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="59.343" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="64.334" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="69.349" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="deleteUserTest" time="76.538" classname="com.qa.gorest.tests.DeleteUserAPITest"/>
  <testcase name="createUserAPIPUTTest_WithConstructor" time="10.339" classname="com.qa.gorest.tests.UpdateUserAPITest"/>
  <testcase name="getUserListAPITest" time="85.556" classname="com.qa.gorest.tests.GetUserAPITest"/>
  <testcase name="createUserAPIPUTTest_WithSetter" time="10.389" classname="com.qa.gorest.tests.UpdateUserAPITest">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at io.restassured.path.json.JsonPath.getString(JsonPath.java:352)
at com.qa.gorest.tests.UpdateUserAPITest.createUserAPIPUTTest_WithSetter(UpdateUserAPITest.java:92)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 18 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createUserAPIPUTTest_WithSetter -->
</testsuite> <!-- Go Rest regression Test -->
